src/views/components/item.jsx

```diff
@@ -51,7 +50,7 @@ export default class Item extends React.Component {
                 <div className="actionArea">
                     <img src="../assets/images/add.svg"/>
                     <img src="../assets/images/edit.svg"/>
-                    <img src="../assets/images/delete.svg"/>
+                    <img onClick={deleteItem.bind(this, this.state.item.address)} src="../assets/images/delete.svg"/>
```

Comment:
There should probably be a confirmation modal, so that we don't have _accidental_ data anomalies (e.g. an item has been deleted whilst it is checked out).

Also, how should we handle deletion of an item when it is currently checked out? Does it get deleted, and we just kinda gloss over the fact that a non-existent item was checked in?